tools/functional-tester/etcd-tester/checks.go

```diff
@@ -87,29 +89,93 @@ func (lc *leaseChecker) Check() error {
 		return err
 	}
 	plog.Infof("checking short lived leases %v", lc.ls.shortLivedLeases.leases)
-	return lc.check(true, lc.ls.shortLivedLeases.leases)
+	return lc.shortLivedLeaseCheck()
+
+}
+
+// shortLivedLeaseCheck verifies the expiry of shortLivedLeases.
+// the logic is follwing:
```

Comment:
I was afraid that the code might not be clear. So I gave a description  to reader what the logic is before he/she start to read. I can remove the comment.